src/Ztm.Data.Entity/Contexts/MainDatabase.cs

```diff
@@ -142,6 +154,11 @@ protected virtual void ConfigureTransaction(EntityTypeBuilder<Transaction> build
             builder.Property(e => e.LockTime).IsRequired();
 
             builder.HasKey(e => e.Hash);
+            builder.HasOne(e => e.ExodusPayload)
```

Comment:
`Transaction > ExodusPayload` does not look right. We should reverse the relationship instead (e.g `ExodusPayload > Transaction`).